.env.demo

```
@@ -0,0 +1,28 @@
+# Demo Environment Configuration
+NODE_ENV=production
+DEMO_MODE=true
+DEMO_PASSWORD=AndyDemo2025!
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEMO_PASSWORD key should go before the NODE_ENV key

(UnorderedKey)